Code/GraphMol/ChemReactions/ReactionWriter.cpp

```diff
@@ -102,6 +102,28 @@ std::string chemicalReactionToRxnToString(
   res += ">";
   res +=
       chemicalReactionTemplatesToString(rxn, RDKit::Product, toSmiles, params);
+
+
+  if (includeCX) {
+    RDKit::MOL_SPTR_VECT mols;
+
+    // Collect reactants, agents, and products into mols vector
+    for (auto type : {RDKit::Reactant, RDKit::Agent, RDKit::Product}) {
+      for (auto begin = getStartIterator(rxn, type);
+           begin != getEndIterator(rxn, type); ++begin) {
+        mols.push_back(*begin);
+      }
+    }
+
+    // Get CX extensions using the new getCXExtensions function
+    // auto flags = RDKit::SmilesWrite::CXSmilesFields::CX_ATOM_PROPS ^ RDKit::SmilesWrite::CXSmilesFields::CX_ALL;
```

Comment:
I think you can remove the `flags` line and this comment

Reply:
Okay, just did.